charts/celeborn/values.yaml

```diff
@@ -33,6 +33,9 @@ masterReplicas: 3
 # worker replicas set on demand, should less than node number
 workerReplicas: 5
 
+hostNetwork: false
+dnsPolicy: Default
```

Comment:
Default means use Host's Dns.
is right that dnspolicy 's default value is ClusterFirst ?

Reply:
Thanks for catching this, fixed.